### PR TITLE
Remove SwaggerUI, couple of minor edits.

### PR DESCRIPTION
**Pull request by davemikolaj**
Real basic doc fixes that I've come across while starting my journey with Shepherd as a developer.

### CODE REVIEW

**Comment by davemikolaj** [comment]
Also wasn't 100% sure of what we want to do with https://github.com/universityofadelaide/shepherd/pull/86/files#diff-710af9c85139525b79a31decd9f92c91L175 as this is UA specific, and developers starting from GitHub aren't going to need this. Confluence, maybe?

File: DEVELOPERS.md [hunk 1]
```diff
@@ -157,17 +154,17 @@ composer install
 ```
 
 ### OpenShift in Docker configuration
-Minishift is really just another layer that you don't need, you can run openshift locally with just
+Minishift is really just another layer that you don't need, you can run OpenShift locally with just
 a couple of tweaks before running ./dsh
-* Install mysql on your local host, or in a docker container listening on 3306 (but it wont work within openshift as shepherd can't talk to it easily).
+* Install mysql on your local host, or in a docker container listening on 3306 (but it wont work within OpenShift as shepherd can't talk to it easily).
```
Comment (pingers) [inline DEVELOPERS.md]: MySQL

Comment (pingers) [reply DEVELOPERS.md]: You can change this bit - remove ` (but it wont work within OpenShift as shepherd can't talk to it easily)`. It works fine for me.

Comment (pingers) [reply DEVELOPERS.md]: *localhost

Comment (davemikolaj) [reply DEVELOPERS.md]: Fixed.

File: DEVELOPERS.md [hunk 2]
```diff
@@ -187,10 +184,12 @@ git remote add shepherd https://github.com/universityofadelaide/shepherd.git
 
 You should end up with something like:
 
-origin	git@gitlab.adelaide.edu.au:web-team/ua-shepherd.git (fetch)
-origin	git@gitlab.adelaide.edu.au:web-team/ua-shepherd.git (push)
-shepherd	https://github.com/universityofadelaide/shepherd.git (fetch)
-shepherd	https://github.com/universityofadelaide/shepherd.git (push)
+```
+origin      git@gitlab.adelaide.edu.au:web-team/ua-shepherd.git (fetch)
+origin	    git@gitlab.adelaide.edu.au:web-team/ua-shepherd.git (push)
```
Comment (pingers) [inline DEVELOPERS.md]: weird indent

**Pull request merged**